src/media/css/elements--nav.styl

```diff
@@ -109,6 +109,7 @@ mkt-nav {
     font-size: 18px;
     height: 100%;
     left: -1 * $nav--width;
+    overflow-x: hidden;
     overflow-y: scroll;
```

Comment:
Should this be `overflow-y: auto` so as not to force a visible scrollbar when it's not necessary on Windows?